tests/pkg_hacl/Makefile

```
@@ -0,0 +1,13 @@
+include ../Makefile.tests_common
+
+BOARD_BLACKLIST := arduino-duemilanove arduino-mega2560 arduino-uno \
+                   jiminy-mega256rfr2 mega-xplained waspmote-pro \
+                   chronos msb-430 msb-430h telosb wsn430-v1_3b wsn430-v1_4 z1
+
+TEST_ON_CI_WHITELIST += native
+
```

It works when increasing the stack size.

```suggestion

CFLAGS += -DTHREAD_STACKSIZE_MAIN=\(5*THREAD_STACKSIZE_DEFAULT\)
```

So same conclusion for me as here https://github.com/RIOT-OS/RIOT/pull/10185/files#r226865550